data/mods/Magiclysm/effects/effects.json

```diff
@@ -507,7 +507,7 @@
     "remove_message": "You no longer feel encumbered.",
     "rating": "good",
     "show_intensity": false,
-    "enchantments": [ { "values": [ { "value": "ATTACK_SPEED", "multiply": -0.25 }, { "value": "MELEE_DAMAGE", "multiply": 0.6 } ] } ]
+    "enchantments": [ { "values": [ { "value": "ATTACK_SPEED", "multiply": 0.25 }, { "value": "MELEE_DAMAGE", "multiply": 0.6 } ] } ]
```

Comment:
One of these two is wrong I think. Wouldn’t this make your speed 1/4 its normal speed instead of 3/4s?  Does that make your melee damage 60% of normal or 160%

Reply:
First about the damage, it adds 60% so 160%, about the speed, it subtracts 25%.
Just tested it and a steel frame went from 267 adjusted moves per attack to 334,  267*1,25 = 333,75 rounded up.